### PR TITLE
Add remaining primary ethereum resolvers

### PR DESCRIPTION
**Pull request by kumavis**
Adds
* ipld-eth-block-list
* ipld-eth-state-trie
* ipld-eth-storage-trie
* ipld-eth-tx-trie

also throws a meaningful error if a specified format is missing

### CODE REVIEW

**Comment by kumavis** [comment]
@diasdavid ready for your review

File: src/index.js [hunk 1]
```diff
@@ -76,8 +84,9 @@ module.exports = class IPLDResolver {
           if (err) {
             return cb(err)
           }
-          const r = this.resolvers[cid.codec]
-          r.resolver.resolve(block, path, (err, result) => {
+          const format = this.resolvers[cid.codec]
+          if (!format) return cb(new Error(`Unknown codec: "${cid.codec}"`))
```
Comment (diasdavid) [inline src/index.js]: please add `{}` for the if clause

File: src/index.js [hunk 2]
```diff
@@ -43,10 +47,14 @@ module.exports = class IPLDResolver {
       }
     }
 
-    // Support by default dag-pb, dag-cbor, and eth-block
+    // Support by default dag-pb, dag-cbor, and eth-*
```
Comment (diasdavid) [inline src/index.js]: I guess we should add this to the readme now, as the list is getting bigger :)

File: package.json [hunk 3]
```diff
@@ -54,6 +54,10 @@
     "ipld-dag-cbor": "0.9.0",
     "ipld-dag-pb": "^0.9.4",
     "ipld-eth-block": "^2.2.1",
+    "ipld-eth-block-list": "^1.0.3",
+    "ipld-eth-state-trie": "^1.0.2",
+    "ipld-eth-storage-trie": "^1.0.0",
+    "ipld-eth-tx-trie": "^1.0.0",
```
Comment (diasdavid) [inline package.json]: Would you be against of having all of this IPLD format for eth on this org? It helps for discoverability, @whyrusleeping was just looking to see what existed and missed that there were all of these already implemented.

Comment (kumavis) [inline package.json]: happy to do that - didnt have auth to add them to the org in the first place

Comment (diasdavid) [inline package.json]: You have it know, right? You should be able to move them.

**Pull request closed without merging**